packages/json-schema/test/extension.test.ts

```diff
@@ -0,0 +1,234 @@
+import { DecoratorContext, Type } from "@typespec/compiler";
+import assert from "assert";
+import { describe, it } from "vitest";
+import { setExtension } from "../src/index.js";
+import { emitSchema } from "./utils.js";
+
+describe("emitting extensions", () => {
```

Comment:
nit, we don't need a top level describe with vitest now as everything is grouped per file